internal/conf/config.go

```diff
@@ -90,8 +90,8 @@ type Settings struct {
 		}
 
 		RTSP struct {
-			Url       string // RTSP stream URL
-			Transport string // RTSP Transport Protocol
+			Transport string   // RTSP Transport Protocol
+			Urls      []string // RTSP stream URL
```

Comment:
@tphakala I think you need to rename this now too, to ```urls``` and maybe set it to [] 

https://github.com/tphakala/birdnet-go/blob/6bc12a5e81ec3bc311cfbcf778819025b6e04a92/internal/conf/defaults.go#L57

Reply:
This is already fixed in latest PR I submitted yesterday

Reply:
I still see the config as a single URL  where I linked on the master branch ..

Reply:
you are correct, missed changes for defaults